engine-tests/src/tests/erc20.rs

```diff
@@ -108,7 +108,7 @@ fn profile_erc20_get_balance() {
     assert!(result.is_ok());
 
     // call costs less than 3 Tgas
-    test_utils::assert_gas_bound(profile.all_gas(), 2);
+    test_utils::assert_gas_bound(profile.all_gas(), 3);
```

Comment:
Why it was increased?

Reply:
I had found the reason actually. I missed a rust compiler flag.